docker/docker-compose.dev.yml

```diff
@@ -16,44 +16,46 @@ services:
     inventree-dev-server:
         container_name: inventree-dev-server
         build:
-            context: .
+            context: ../
             target: dev
+            dockerfile: docker/Dockerfile
+        environment:
+            INVENTREE_DEV_VENV: "True"
         ports:
             - 8000:8000
+        user: root
         volumes:
-            # Ensure you specify the location of the 'src' directory at the end of this file
-            - src:/home/inventree
+            # Ensure you specify the location of the 'INVENTREE_SRC_DIR' in the dev-config.env file
+            - ${INVENTREE_SRC_DIR:?inventree_src_dir}:/home/inventree
         env_file:
             # Environment variables required for the dev server are configured in dev-config.env
             - dev-config.env
 
         restart: unless-stopped
 
+        command: ["/init-server.sh", "python3", "manage.py", "runserver", "0.0.0.0:8000"]
```

Comment:
If we specify an "entrypoint" and "cmd" under the "dev" target in the dockerfile, do we need to specify the command here in the docker-compose file?

It would seem to me that coding the correct entrypoints / commands into the dockerfile targets means that we can write simpler docker-compose files?

Reply:
these are more placeholders for the following PR. Some of these variables like the port could/should? be interpolated by the compose file. 

I should have updated the CMD entries in the Dockerfile to match those currently present in the docker-compose.yml file .